helm-charts/charts/kube-starrocks/templates/starrocks-operator/deployment.yaml

```diff
@@ -46,6 +54,15 @@ spec:
         # TODO(user): Configure the resources accordingly based on the project requirements.
         # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
-{{ toYaml .Values.starrocksOperator.resources | indent 10 }}
+          {{- toYaml .Values.starrocksOperator.resources | nindent 10 }}
       serviceAccountName: {{ template "kube-starrocks.operator.serviceAccountName" . }}
       terminationGracePeriodSeconds: 10
+      {{- if .Values.starrocksOperator.nodeSelector }}
+      nodeSelector:
+        {{- toYaml .Values.starrocksOperator.nodeSelector | nindent 8 }}
```

Comment:
is there any guideline or code style that those control statement should be written with no indention and those `toYaml` content output is controlled by `nindent`?

Reply:
If you  execute `helm create foo`, a helm chart demo will be created, and I follow this code style. e.g. 

```
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}

```